example/committee.yml

```diff
@@ -0,0 +1,20 @@
+- role: General Chairs
+  members:
+  - name: Bonnie Webber
```

Comment:
The same pattern of `attendee` as a `name` + `institution` appears in a bunch of different places. 
Structs like this can be better expressed as dataclasses in Python. Having an API like this could make it easier on users of this pub interface, but the tradeoff is locking into Python as a language.
```py
Attendee(
    name="Bonnie Webber",
    institution="University of Edinburgh, UK",
)
```